api/src/db/migrations/v145_remove_haserror_field_from_licensedetails.ts

```diff
@@ -0,0 +1,833 @@
+import {
+  v144Business,
+  v144LicenseDetails,
+  v144LicenseName,
+  v144UserData,
+} from "@db/migrations/v144_add_raffle_bingo_games";
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v144_to_v145 = (v144Data: v144UserData): v145UserData => {
+  return {
+    ...v144Data,
+    businesses: Object.fromEntries(
+      Object.values(v144Data.businesses)
+        .map((business: v144Business) => migrate_v144Business_to_v145Business(business))
+        .map((currBusiness: v145Business) => [currBusiness.id, currBusiness])
+    ),
+    version: 145,
+  } as v145UserData;
+};
+
+export const migrate_v144Business_to_v145Business = (business: v144Business): v145Business => {
+  const newLicenseData = deleteFieldFromInnerObjects(business.licenseData?.licenses);
+
+  return {
+    ...business,
+    profileData: {
+      ...business.profileData,
+      licenseData: {
+        licenses: newLicenseData,
+        ...business.licenseData,
+      },
+    },
+  } as v145Business;
+};
+
+export const deleteFieldFromInnerObjects = (
+  v144Licenses: Partial<Record<v144LicenseName, v144LicenseDetails>> | undefined
+): Partial<Record<v145LicenseName, v145LicenseDetails>> | undefined => {
```

Comment:
pebble - v144Licenses exist in the other file and can be imported, you should reference v145Licenses since it exist... Not Partial<Record<v144LicenseName, v144LicenseDetails>>